### DISCUSSION TITLE
Project Showcase

### DISCUSSION THREAD

**Discussion by brizzbuzz** [bkbnio/kompendium#444] [Show and tell]
Showcase for teams who have used Kompendium to show off their amazing documentation 🧙✨

**Comment by mjovanc** [comment]
We, the Kryptokrona project, with our Kryptokrona API are using it: [https://github.com/kryptokrona/kryptokrona-api](https://github.com/kryptokrona/kryptokrona-api) It's awesome with a very easy setup! Check it out!
